src/app/fdctl/run/tiles/fd_store_int.c

```diff
@@ -588,7 +625,7 @@ unprivileged_init( fd_topo_t *      topo,
     }
     FD_LOG_NOTICE(( "finished blockstore_wksp restore %s", tile->store_int.blockstore_restore ));
     fd_wksp_tag_query_info_t info;
-    ulong tag = FD_BLOCKSTORE_MAGIC;
```

Comment:
remove hardcoding

Reply:
Right, I have fixed it as below and, when I run it, it does give me `tag==1` as expected.